src/App.tsx

```diff
@@ -1,23 +1,25 @@
-import { Authenticator } from "@aws-amplify/ui-react";
+// import { Authenticator } from "@aws-amplify/ui-react";
 import "@aws-amplify/ui-react/styles.css";
 import { Auth } from "aws-amplify";
 import "./App.css";
+import Home from "./components/Home";
 import awsconfig from "./aws-exports";
 // import configuraiton of AWS authentication
 Auth.configure(awsconfig);
 
 function App() {
   return (
-    <Authenticator>
-      {({ signOut, user }: any) => (
-        <div>
-          <p>Welcome to wilshire, {user.username}</p>
-          <button type="button" onClick={signOut}>
-            Sign out
-          </button>
-        </div>
-      )}
-    </Authenticator>
+    <Home />
+    // <Authenticator>
+    //   {({ signOut, user }: any) => (
```

Comment:
This isn't quite what we want. The Authenticator tags should stay here. From further research, we should keep the anonymous component but we can still add all the inside stuff in the new Home component